grpcurl.go

```diff
@@ -161,6 +163,36 @@ func MetadataFromHeaders(headers []string) metadata.MD {
 	return md
 }
 
+var envVarRegex = regexp.MustCompile(`\${\w+}`)
+
+// ExpandHeaders expands environmental variables contained in the header string.
+// If no corresponding environmental variable is found an error is thrown.
+// TODO: Add escaping for `${`
+func ExpandHeaders(headers []string) ([]string, error) {
+	expandedHeaders := make([]string, len(headers))
+	for idx, header := range headers {
+		if header != "" {
```

Comment:
Instead of deepening the logic with indentation under an `if` block, how about invert the condition:

```go
if header == "" {
    continue
}
```

Similar for below:

```go
if len(results) == 0 {
    expandedHeaders[idx] = headers[idx]
    continue
}

```